server/swissvoice/static/css/landingpage.css

```diff
@@ -10,15 +10,20 @@
 }
 
 .jumbotron {
-  background-color: #baeeff; /*color action-section*/
+  background-color: #d6f1ff; /*color action-section*/
 }
 
 #main-schreiben-btn {
   font-size: 1.5rem;
-  background-color:  #007fff; /*color of button in action-section*/
+  background-color:  #63aeff; /*color of button in action-section*/
   border-color: transparent;
 }
 
+#main-schreiben-btn:hover {
```

Comment:
why the denglish?

Reply:
Caus this button will (in the futur) lead to a page were only the "schreiben" will take place.